diff --git a/node_modules/@typescript/dom-lib-generator/inputfiles/overridingTypes.jsonc b/node_modules/@typescript/dom-lib-generator/inputfiles/overridingTypes.jsonc
index 7c11453..c7f0bbb 100644
--- a/node_modules/@typescript/dom-lib-generator/inputfiles/overridingTypes.jsonc
+++ b/node_modules/@typescript/dom-lib-generator/inputfiles/overridingTypes.jsonc
@@ -4035,6 +4035,23 @@
                     }
                 ]
             },
+            {
+                "name": "ImageDataArray",
+                "typeParameters": [
+                    {
+                        "name": "ArrayBuffer"
+                    }
+                ],
+                "type": [
+                    {
+                        "type": "Uint8ClampedArray"
+                    },
+                    {
+                        "type": "Float16Array"
+                    }
+                ],
+                "overrideType": "Uint8ClampedArray<ArrayBuffer> | Float16Array<ArrayBuffer>"
+            },
             {
                 "name": "ReadableStreamReader",
                 "typeParameters": [
diff --git a/node_modules/@typescript/dom-lib-generator/inputfiles/removedTypes.jsonc b/node_modules/@typescript/dom-lib-generator/inputfiles/removedTypes.jsonc
index 8089030..d93ee40 100644
--- a/node_modules/@typescript/dom-lib-generator/inputfiles/removedTypes.jsonc
+++ b/node_modules/@typescript/dom-lib-generator/inputfiles/removedTypes.jsonc
@@ -410,13 +410,6 @@
                     }
                 }
             },
-            "ImageDataSettings": {
-                "members": {
-                    "member": {
-                        "pixelFormat": null // Blink experimental only as of 2025-04
-                    }
-                }
-            },
             "IntersectionObserverInit": {
                 "members": {
                     "member": {
